templates/install.rdf.mustache

```diff
@@ -7,10 +7,10 @@
     <em:multiprocessCompatible>{{addon.multiprocessCompatible}}</em:multiprocessCompatible>
     <em:hasEmbeddedWebExtension>{{addon.hasEmbeddedWebExtension}}</em:hasEmbeddedWebExtension>
     <em:unpack>false</em:unpack>
-    <em:version>{{addon.version}}</em:version>
+    <em:version>{{version}}</em:version>
     <em:name>{{addon.name}}</em:name>
-    <em:description>{{addon.description}}</em:description>
-    <em:creator>{{addon.creator}}</em:creator>
+    <em:description>{{description}}</em:description>
+    <em:creator>{{author}}</em:creator>
```

Comment:
These are the handfull of configs that we can proxy from the package.json `version`, `author`, and `description` instead of duplicating those configs in 2 places. This should [ideally] make bumping version numbers less tricky since they only need to be set in a single place.